src/filesystem.cpp

```diff
@@ -735,17 +740,21 @@ bfs::path apple_userdata(const std::string& newprefdir)
 	if(temp.empty()) {
 #ifdef PREFERENCES_DIR
 		temp = PREFERENCES_DIR;
+		DBG_FS << "userdata using PREFERENCES_DIR '" << PREFERENCES_DIR;
```

Comment:
There's an opening quote but no closing quote. It doesn't really matter which, but I'd say put both or neither.

Reply:
Whoops, fixed.